manager/install.sh

```diff
@@ -378,7 +378,7 @@ function remove_nodegroups() {
 
   echo "￮ removing nodegroup(s) from the cluster ..."
   python generate_eks.py $CORTEX_CLUSTER_CONFIG_FILE manifests/ami.json --remove-eks-node-groups="$eks_nodegroup_names" > /workspace/nodegroups.yaml
-  eksctl delete nodegroup --timeout=$EKSCTL_NODEGROUP_TIMEOUT --approve -f /workspace/nodegroups.yaml
+  eksctl delete nodegroup --timeout=$EKSCTL_NODEGROUP_TIMEOUT --drain=false --approve -f /workspace/nodegroups.yaml
```

Comment:
Has this been tested with an API that was running on node group that was about to get removed? I believe the drain is important.

Reply:
Yes, `--drain=false` is preferred. I was having issues with it on `eksctl` version 0.73, but since I've downgraded it to v0.67 it might work now! I'll give it a try

Reply:
Yes it's working as expected now, so I'll revert it, thanks for bringing it up! We'll just have to remember to test this again next time we upgrade `eksctl`

Reply:
I see. Thanks for the confirmation on this!